install/kubernetes/cilium/templates/cilium-configmap.yaml

```diff
@@ -1355,3 +1355,14 @@ data:
   config: |-
 {{ toJson .Values.ipMasqAgent.config | indent 4 }}
 {{- end }}
+---
+{{- if and .Values.clustermesh.config.enabled }}
```

Comment:
We don't need to create a new configmap. We can use the same one that is already used nu clustermes. This would provide two benefits:
- whenever new cluster is added, we will act on a single event to start synchronizing the new cluster and resolve fqdn for it - an additional benefit of not having a problem in case of two configmaps being in a different states
- We won't need to add additional permissions for the agent to be able to watch this configmap

Reply:
I've rewritten the resolver to load entries from the clustermesh secret. Please take another look.